l10n_ch_import_camt054/models/custom_parser.py

```diff
@@ -92,6 +96,12 @@ def parse_entry(self, ns, node):
         for node in details_nodes:
             transaction = transaction_base.copy()
             self.parse_transaction_details(ns, node, transaction)
+
+            # If there is a AddtlNtryInf, then we do the concatenate
+            if addtl_ntry_inf:
+                transaction['name'] = transaction['name'] + \
+                                      ' - [' + addtl_ntry_inf[0].text + ']'
```

Comment:
I know flake8 is a real pain. This line needs only one tab indent relative to the previous. By the way you can also shorten by doing `transaction['name'] += ' - [{}]'.format(addtl_ntry_inf[0].text)`